src/scripting_api/field.js

```diff
@@ -267,6 +272,10 @@ class Field extends PDFObject {
     }
   }
 
+  _getValue() {
+    return this._originalValue !== undefined ? this._originalValue : this.value;
```

Comment:
Nit: Based on the code above it doesn't look like `this._originalValue === null` can even happen, so how about simply:
```suggestion
    return this._originalValue ?? this.value;
```